src/openzaak/components/zaken/api/serializers/zaken.py

```diff
@@ -1085,6 +1085,19 @@ def run_validators(self, value):
         return super().run_validators(value)
 
 
+class ZaakInformatieObjectReadOnlySerializer(ZaakInformatieObjectSerializer):
+    informatieobject = EnkelvoudigInformatieObjectField(
+        max_length=1000,
+        min_length=1,
+        help_text=get_help_text("zaken.ZaakInformatieObject", "informatieobject"),
+        required=False,
+        read_only=True,
+    )
+
+    class Meta(ZaakInformatieObjectSerializer.Meta):
+        validators = []
```

Comment:
This means that the validators from `ZaakInformatieObjectSerializer` like UniqueTogetherValidator on zaak/informatieobject are not applied, is that intentional?